docs/oauth2.rst

```diff
@@ -119,12 +119,12 @@ For End Users: Using OAuth Client ID
 This is the case where your application or a script is accessing spreadsheets on behalf of an end user. When you use this scenario, your application or a script will ask the end user (or yourself if you're running it) to grant access to the user's data.
 
 1. :ref:`enable-api-access` if you haven't done it yet.
-2. Go to "APIs & Services > OAuth Consent Screen." Click the button for "Configure Consent Screen" and follow the directions to give your app a name; you don't need to fill out anything else on that screen. Click Save. 
+2. Go to "APIs & Services > OAuth Consent Screen." Click the button for "Configure Consent Screen" and follow the directions to give your app a name; you don't need to fill out anything else on that screen. Click Save.
 3. Go to "APIs & Services > Credentials"
 4. Click "+ Create credentials" at the top, then select "OAuth client ID".
 5. Select "Desktop app", name the credentials and click "Create". Click "Ok" in the "OAuth client created" popup.
 6. Download the credentials by clicking the Download JSON button in "OAuth 2.0 Client IDs" section.
-7. Move the downloaded file to ``~/.config/gspread/credentials.json``. Windows users should put this file to ``%APPDATA%\gspread\credentials.json``.
+7. Move the downloaded file to ``~/.config/gspread/authorized_user.json``. Windows users should put this file to ``%APPDATA%\gspread\authorized_user.json``.
```

Comment:
I see, so I changed this because I thought it was a mistake since the constant `DEFAULT_AUTHORIZED_USER_FILENAME` points to `authorized_user.json` https://github.com/burnash/gspread/blob/master/gspread/auth.py#L52-L54

But I may be mistaken... I don't have the ability to generate this kind of file on my personal account so I couldn't really test it. If you think it's not a mistake I'm totally happy to change it back, just let me know.

Reply:
`authorized_user.json` is generated during OAuth flow. `InstalledAppFlow.from_client_secrets_file` reads `DEFAULT_CREDENTIALS_FILENAME` which is `credentials.json`. Please see `auth.py` for details.

Reply:
If you rename `credentials.json` to `authorized_user.json` you get `FileNotFoundError`.

Reply:
I see I see! Ok reverting this change then.